hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -200,6 +203,19 @@
       .defaultValue("true")
       .withDocumentation("Similar to " + BLOOM_INDEX_UPDATE_PARTITION_PATH_ENABLE + ", but for simple index.");
 
+  // ***** Bucket Index Configs *****
+  public static final ConfigProperty<Integer> BUCKET_INDEX_BUCKET_NUM = ConfigProperty
+      .key("hoodie.index.bucket.num")
+      .defaultValue(-1)
```

Comment:
@minihippo here why using -1 as the default value? can we use a recommended value such as 128/256 or other value for BUCKET index? because I think users may not know how to set the value.

Reply:
+1